src/rdb.c

```diff
@@ -3659,6 +3674,11 @@ void bgsaveCommand(client *c) {
     } else if (hasActiveChildProcess() || server.in_exec) {
         if (schedule || server.in_exec) {
             server.rdb_bgsave_scheduled = 1;
+            if (schedule) {
+                serverLog(LL_NOTICE, "Background saving scheduled due to user request");
+            } else {
+                serverLog(LL_NOTICE, "Background saving scheduled to run after transaction execution");
+            }
```

Comment:
Maybe this is enough? i add a same line for aof, see #826
```suggestion
            serverLog(LL_NOTICE, "Background saving scheduled.");
```

Reply:
I think it is clearer to explicitly state why the scheduling was done. do you think this is TMI?

Reply:
yes, i think this is minor. this is a admin command, so i think it's enough to keep it simple

Reply:
I agree, but for inspecting logs from distant past might be beneficial.